app/main.py

```diff
@@ -1,6 +1,18 @@
 from typing import Callable
+import functools
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    caches = {}
+
+    @functools.wraps(func)
+    def inner(*args) -> None:
```

Comment:
```suggestion
    def inner(*args, **kwargs) -> None:
```
You should consider that the function can accept keyword arguments as well.